test/parallel/test-fs-read-stream-encoding.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const stream = require('stream');
+const encoding = 'base64';
+
+const example = path.join(common.fixturesDir, 'x.txt');
+const assertStream = new stream.Writable({
+  write: function(chunk, enc, next) {
+    const expected = new Buffer('xyz');
```

Comment:
passed. 
I guess if I choose 'utf8' as encoding type, I should change `'xyz\n\n'`;